pkgs/applications/graphics/eddy/default.nix

```diff
@@ -0,0 +1,42 @@
+{ python37Packages
+, fetchFromGitHub
+, lib
+, jre
+, qt5
+}:
+
+python37Packages.buildPythonApplication rec {
+  pname = "eddy";
+  version = "v1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "obdasystems";
+    repo = pname;
+    rev = version;
+    sha256 = "12j77bbva5py9bd57c80cmjvf8vll40h19n81h16lvv2r2r7jynh";
+  };
+
+  propagatedBuildInputs = [
+    qt5.full
+    python37Packages.setuptools
+    python37Packages.rfc3987
+    python37Packages.JPype1
+    python37Packages.pyqt5
+  ];
+
+  doCheck = false;
+
+  HOME = "/tmp";
```

Comment:
Please export this env where needed.

Reply:
What do you mean? If I don't set that env, the package can't be build.

Reply:
```
preBuild = ''
  export HOME=/tmp
```

Reply:
Resolved